app/controllers/api/v1/latest_photos_controller.rb

```diff
@@ -1,26 +1,21 @@
 class Api::V1::LatestPhotosController < ApplicationController
   def index
-    def index
-      @rover = Rover.find_by name: params[:rover_id].titleize
-      if @rover
-        render json: search_photos, each_serializer: PhotoSerializer, root: :latest_photos
-      else
-        render json: { errors: "Invalid Rover Name" }, status: :bad_request
-      end
-    end
-  end
+    rover = Rover.find_by name: params[:rover_id].titleize
 
-  private
+    if rover then
```

Comment:
`then` is not required here, and isn't really used commonly in Ruby.

Reply:
Thanks for catching this! I took it out :)